lib/platform/github/index.ts

```diff
@@ -1671,14 +1671,24 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
       }
     }
   }`;
-  let alerts: VulnerabilityAlert[] = [];
+  let vulnerabilityAlerts: {
+    node: VulnerabilityAlert;
+  }[];
   try {
-    const vulnerabilityAlerts = await githubApi.queryRepoField<{
+    vulnerabilityAlerts = await githubApi.queryRepoField<{
       node: VulnerabilityAlert;
     }>(query, 'vulnerabilityAlerts', {
       paginate: false,
       acceptHeader: 'application/vnd.github.vixen-preview+json',
     });
+  } catch (err) {
+    logger.warn(
+      { err },
+      'Error retrieving vulnerability alerts. Please ensure permissions have been granted.'
```

Comment:
Can we explain where to grant these permissions? Or link to the docs (if we have those)?

The current message lacks enough information to resolve the problem, at least for me. I would not know what to do if I got this warning... 😄 🙈 

I searched the docs for "vulnerability" and I got this documentation:
https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts
which does not mention permissions at all.